readme.txt

```diff
@@ -82,11 +96,7 @@ Download the WooCommerce Plugin now, [register at shipcloud.io](http://bit.ly/sh
 1. Upload the folder `shipcloud-for-woocommerce` to the `/wp-content/plugins/` directory
 2. Activate the plugin through the 'Plugins' menu in WordPress
 3. [Register an account](http://bit.ly/shipcloud-for-woocommerce-en) with shipcloud
-4. Go to the `WooCommerce/Settings/Shipping` tab in the WordPress Admin, select `shipcloud`, add your API Key and put in the rest of the settings
-5. Be sure that you have setup the shipment related settings (length, height, width and weight) in your products ...
-6. ... or if you are using Shipping Classes, add the settings there
-7. After setting up, your new orders will have a box called `shipcloud shipment center`
-8. There you can calculate prices for your parcels as well as creating & downloading shipping labels
+4. Follow this simple instruction video: https://youtu.be/HE3jow15x8c
```

Comment:
@andrecedik Those videos are held in German. But I guess the only usage will be in Germany, right?